### PR TITLE
989 Create MembershipPage

### LINKED ISSUE #989: Create UI of MembershipPage
Add the contents of the MembershipPage: navbar, footer, text ...

### PR DESCRIPTION
**Pull request by amaliejvik**
- Create MembershipPage that can be reached from Navbar > Information > Membership. 
- Set up correct routing.
- The information on MembershipPage is displayed using TextItems, providing dark-mode- and translation-functionality (added in seed script, so one should seed in the branch before opening localhost)

NB: The url link to Luka is not working, because the page has not been set up yet. 

Closes #989 #990 #991

### CODE REVIEW

**Review by snorrekr — APPROVED** [review]

**Review by Snorre98 — APPROVED** [review]

**Comment by amaliejvik** [comment]
vet ikke hvorfor junior sin review ikke teller som code owner review?
tror jeg trenger en av deres approve for å få merget hehe 😇  @robines @tiniusp

**Review by tiniuspre — APPROVED** [review]

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import 'src/mixins';
+
+@import 'src/constants';
+
+.link {
+  color: $red-samf;
+  font-family: sans-serif;
```
Comment (robines) [inline frontend/src/Pages/MembershipPage/MembershipPage.module.scss:7]: Skal ikke trenge å spesifisere font-family her, det skal arve font automatisk.

```suggestion
```

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.header {
+  align-content: center;
+  color: black;
+  font-family: sans-serif;
```
Comment (robines) [inline frontend/src/Pages/MembershipPage/MembershipPage.module.scss:13]: Se andre kommentar om `font-family`

```suggestion
```

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  @include theme-dark {
+    color: $theme-dark-color;
+  }
+}
+
+.header2 {
+  font-weight: 550;
+  font-size: 1.5em;
+  font-family: sans-serif;
```
Comment (robines) [inline frontend/src/Pages/MembershipPage/MembershipPage.module.scss:26]: Se andre kommentar om `font-family`

```suggestion
```

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  margin-top: 0.3em;
+  padding-bottom: 0.5em;
+}
+
+.info {
+  text-align: left;
+  box-sizing: inherit;
+  place-content: center;
+  font-size: 20px;
+  font-family: sans-serif;
```
Comment (robines) [inline frontend/src/Pages/MembershipPage/MembershipPage.module.scss:36]: Se andre kommentar om `font-family`

```suggestion
```

File: frontend/src/Pages/MembershipPage/MembershipPage.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        </div>
+        <div className={styles.info}>
+          <h2 className={styles.header2}>{useTextItem(TextItem.register_card)}</h2>
+          <div>{useTextItem(TextItem.register_card_text)}</div>
+        </div>
+        <div className={styles.info}>
+          <h2 className={styles.header2}>{useTextItem(TextItem.laws_and_statutes_header)}</h2>
+          <div>
+            {useTextItem(TextItem.laws_and_statutes_text)}{' '}
+            <a className={styles.link} href={ROUTES.other.lover_og_statutter}>
```
Comment (robines) [inline frontend/src/Pages/MembershipPage/MembershipPage.tsx:89]:
```suggestion
            <a className={styles.link} href={ROUTES.other.laws_and_statutes}>
```

File: frontend/src/routes/other.ts [hunk 6]
```diff
@@ -7,4 +7,6 @@ export const ROUTES_OTHER = {
     '          .42220330892942!2d10.3953201100997!3f352.98823313957973!4f2.236618333891684!5f0.7820865974627469',
   maps_lyche: 'https://maps.app.goo.gl/z98jeRNVqQUPRWsv9',
   lyche_facebook: 'https://www.facebook.com/samfundet.lyche',
+  akademika: 'https://www.akademika.no/butikker',
+  lover_og_statutter: 'http://raadet.samfundet.no/lover.html',
```
Comment (robines) [inline frontend/src/routes/other.ts:11]:
```suggestion
  laws_and_statutes: 'http://raadet.samfundet.no/lover.html',
```

**Review by robines — CHANGES_REQUESTED** [review]

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss (hunk 1 above)
Comment (amaliejvik) [reply frontend/src/Pages/MembershipPage/MembershipPage.module.scss]: fikset:)

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss (hunk 4 above)
Comment (amaliejvik) [reply frontend/src/Pages/MembershipPage/MembershipPage.module.scss]: fikset:)

File: frontend/src/Pages/MembershipPage/MembershipPage.tsx (hunk 5 above)
Comment (amaliejvik) [reply frontend/src/Pages/MembershipPage/MembershipPage.tsx]: fikset:)

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss (hunk 4 above)
Comment (amaliejvik) [reply frontend/src/Pages/MembershipPage/MembershipPage.module.scss]: fikset:)

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss (hunk 3 above)
Comment (amaliejvik) [reply frontend/src/Pages/MembershipPage/MembershipPage.module.scss]: fikset:)

File: frontend/src/Pages/MembershipPage/MembershipPage.module.scss (hunk 2 above)
Comment (amaliejvik) [reply frontend/src/Pages/MembershipPage/MembershipPage.module.scss]: fikset:)

**Review by robines — APPROVED** [review]

**Pull request merged**